README.md

```diff
@@ -161,34 +77,25 @@ The following ports are exposed by this container:
 
 | Port | Purpose        |
 |------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| 8080 | Con-PCA Tasks  |
 
 ## Environment variables ##
 
 ### Required ###
 
-There are no required environment variables.
+There are no required environment variables as of now.
```

Comment:
I suggest just leaving this as it in the skeleton, since your change doesn't really add anything.

Reply:
@jsf9k resolved with [6a7524e](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/6a7524ea2170f8ceb79a123a7de4f04873f1b822)